api/ai/priority_algorithm/mutations/random_team_size_change.py

```diff
@@ -0,0 +1,65 @@
+import random
+from typing import Dict, List
+
+from api.ai.interfaces.team_generation_options import TeamGenerationOptions
+from api.ai.priority_algorithm.custom_dataclasses import PriorityTeamSet
+from api.ai.priority_algorithm.mutations.interfaces import Mutation
+from api.ai.priority_algorithm.mutations.utils import get_available_priority_teams
+from api.ai.priority_algorithm.priority.interfaces import Priority
+from api.dataclasses.student import Student
+
+
+class RandomTeamSizeMutation(Mutation):
+    def __init__(self, num_mutations: int = 1, number_of_teams: int = 2):
```

Comment:
you can just do *args, **kwargs so you don't have to update every mutation each time we change Mutation